tests/src/ExistingSiteJavascript/UpdatePageTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Drupal\Tests\acquia_cms\ExistingSiteJavascript;
+
+use weitzman\DrupalTestTraits\ExistingSiteSelenium2DriverTestBase;
+
+/**
+ * Tests update.php page as per ACMS new design.
+ *
+ * @group acquia_cms
+ * @group low_risk
+ * @group pr
+ * @group push
+ */
+class UpdatePageTest extends ExistingSiteSelenium2DriverTestBase {
+
+  /**
+   * Tests update page design.
+   *
+   * @throws \Behat\Mink\Exception\ElementNotFoundException
+   * @throws \Behat\Mink\Exception\ResponseTextException
+   * @throws \Drupal\Core\Entity\EntityStorageException
+   */
+  public function testUpdatePage() {
+    $account = $this->createUser();
```

Comment:
@chandan-singh7929 Not sure if we need to delete the user & role both or does it handle that automatically ?

Reply:
We don’t have to do that, it automatically handles that.